src/core/issues-notes.js

```diff
@@ -139,28 +139,28 @@ export function run(conf, doc, cb) {
   var $ins = $(".issue, .note, .warning, .ednote"),
     ghIssues = {},
     issueBase = conf.issueBase;
+  async function asyncFetch() {
+    try {
+      const json = await ghFetch(conf.githubAPI);
+      const issueUrl = issueBase || json.html_url + "/issues/";
+      const issues = await fetchIndex(json.issues_url, {
+        // Get back HTML content instead of markdown
+        // See: https://developer.github.com/v3/media/
+        Accept: "application/vnd.github.v3.html+json",
+      });
+      ghIssues = issues.reduce((issuesObj, issue) => {
```

Comment:
Re-assinging `ghIssues` seems redundant, as we already initialized it to an empty object above.

Reply:
you want me to over ride the global object inside reduce?

This doesn't seem nice as recuce should have it's own local variable inside it

Reply:
> you want me to over ride the global object inside reduce? 

Consider, it's "reducing prop/value into", but not overriding. ghIssues remains the same reference.

Reply:
thanks, made the changes